.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -119,7 +119,7 @@ To help with the load of incoming pull requests:
 
 - [ ] I have reviewed two other [open pull requests][prs] in this repository.
 
-[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-asc+-review%3Aapproved
+[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-asc+-review%3Aapproved+-review:changes%3Arequested
```

Comment:
Link doesn't seem to work here, nor am I able to figure out a working version. Bug in GitHub maybe?

Reply:
It "works", but will only filter out where the only review state is "changes requested", if it contains other review comments the PR will still show in the search